articles/connections/social/apple.md

```diff
@@ -4,63 +4,65 @@ connection: Apple
 :::
 
-## 1. Set up your app in your Apple Developer Account
+## Prerequisites
 
-Once your Developer Account is set up, you can follow the instructions in the [Apple Setup Guide](/connections/apple-setup) to get your app set up.
+Before you add support for SIWA to your app, you'll need:
 
-While setting up your app, make sure you save the following items for later:
+* An [Apple Developer](https://developer.apple.com/programs/) account, which is a paid account with Apple. (There is no free trial available unless you are part of their [iOS Developer University Program](https://developer.apple.com/support/compare-memberships/).).
+* A domain to point to and an internet-accessible server where you will run the app that responds on behalf of this domain. You will also need to configure this server with a TLS certificate. Apple won't accept unsecured HTTP connections. 
 
-* The Client ID (the Service ID)
-* The Client Secret Signing Key
-* The Apple Team ID
-* The Client Signing Key ID (optional)
+* A [Custom Domain](/custom-domains) set up on your Auth0 tenant for domain verification with Apple. Custom domains are not strictly required to use SIWA authorization. Domain validation is required for sending emails to private Apple addresses in native and web apps. To use the [Email Relay Service](/connections/apple-siwa/configure-email-relay-service), you will also need to configure your domain with Sender Policy Framework (SPF) DNS TXT records.
 
-When setting up the app, your **Return URL** should be in the following format: `https://<YOUR CUSTOM DOMAIN>/login/callback`. If you have any issues, refer to the [Apple Documentation](https://developer.apple.com/sign-in-with-apple/get-started/).
+## How it works
 
-## 2. Create and enable a connection in Auth0
+Once you have registered your application with Apple and configured your application connection settings in Auth0 to use the IDs and keys obtained from Apple, your users can sign in to your applications using their Apple IDs and passwords. 
 
-Once you have the credentials you need from your Apple developer account, go to [**Connections** > **Social**](${manage_url}) in the Dashboard, and click on the **Apple** connection.
+![Apple Sign In with Apple Setup Flow](/media/articles/connections/social/apple/apple-siwa-setup-flow.png)
 
-Fill in the Client ID (Services ID), Client Secret Signing Key, the Team ID, and the Client Signing Key ID (if you have it) here. You can also fill in the Key ID, but this is optional, as Apple will accept the key without the ID.
+When your users return to your app, you can log them in automatically if their login session is still valid. To do this you will:
 
-![Apple Connection Settings](/media/articles/connections/social/apple/apple_connection.png)
+* Call `ASAuthorizationAppleIDProvider.getCredentialState` to make sure the user is still authorized.
+* Retrieve Auth0 credentials.
```

Comment:
This feels strange here.  Above ^, we don't go into detail around the code behind how a user performs the initial login.  Why do we do so here?  I think either both of these should go into that level of detail, or neither.  Feels like apples and oranges.

Reply:
Agreed. I'll remove the details from the second case and just summarize it.